pkg/webhook/validation/uservalidation.go

```diff
@@ -82,27 +84,25 @@ func ValidateMemberClusterUpdate(currentObj, oldObj client.Object, whiteListedUs
 
 // ValidateInternalMemberClusterUpdate checks to see if user is allowed to update argued internal member cluster resource.
 func ValidateInternalMemberClusterUpdate(ctx context.Context, client client.Client, imc fleetv1alpha1.InternalMemberCluster, whiteListedUsers []string, userInfo authenticationv1.UserInfo, subResource string) admission.Response {
-	imcKind := "InternalMemberCluster"
 	namespacedName := types.NamespacedName{Name: imc.Name, Namespace: imc.Namespace}
 	if subResource == "status" {
```

Comment:
as I mentioned in a previous PR, this logic should be in the handle function, it's just an overhead.